packages/server/app.ts

```diff
@@ -230,7 +230,8 @@ function buildApolloSubscriptionServer(
         }
 
         return baseParams
-      }
+      },
+      keepAlive: 30000 //milliseconds. Loadbalancers may close the connection after inactivity. e.g. nginx default is 60000ms.
```

Comment:
isn't the goal here, to keep the conn alive for longer than 60 secs, that is respected by nginx too?